### ISSUE TITLE
html: background color of the body-tag is respected only for the first snippet

### ISSUE THREAD

**Issue by ralf3u** [xiota/geany-preview#21]
```
<body style="background:gray">
```
in preview.conf: `snippet_html=true`

tested with `extra_css=extra-media.css` and also tested with `extra_css=disable`

**Comment by xiota** [comment]
That is because the beginning of the document with the style info gets cut off.

**Comment by ralf3u** [comment]
>That is because the beginning of the document with the style info gets cut off.

So, instead of the code above, the alternative is to add the background color to the css in extra_css in preview.conf that works for all snippets. I'm fine with a wontfix-label on this issue that could be closed.

**Comment by xiota** [comment]
>  I'm fine with a wontfix-label on this issue that could be closed.

I'll leave this open because I may look at it eventually.  It's just that restructuring the plugin to make it more manageable would have to be done first.

**Comment by ralf3u** [comment]
>It's just that restructuring the plugin to make it more manageable would have to be done first.

@xiota You don't need to fix issues for the actual plugin. I can wait for the redesigned plugin, because all important issues for my needs are fixed, and because I will not start to code in the next ten months. But I still publish issues, so you are informed, so you can maybe think about it to solve it in the redesigned plugin, and so you can test the issue later in the redesigned plugin.

If I recognize that it would be good to fix an issue soon, then I would ask you if it could be fixed in the next weeks.

**Comment by xiota** [comment]
> You don't need to fix issues for the actual plugin. I can wait for the redesigned plugin, because all important issues for my needs are fixed...  But I still publish issues, so you are informed...

Thanks for reporting.  Just wanted to let you know you're not being ignored.

> If I recognize that it would be good to fix an issue soon, then I would ask you if it could be fixed in the next weeks.

That is helpful, like with the shaking problem.  So I know what to prioritize.

**Comment by xiota** [comment]
Closing because snippets feature has been removed from the 0.2.x rewrite.
